frontend/.eslintrc.json

```diff
@@ -34,6 +34,9 @@
                 "tsx": "never"
             }
         ],
+        "jsx-a11y/label-has-associated-control": [ 2, {
+            "controlComponents": ["CheckboxInput"]
+        }],
```

Comment:
Thanks for adding a checkbox input for the "Hidden" field on test cases -- can you note what this code is adding here?

Reply:
When using a label with an input, you have to nest the input inside the label as well as set the `htmlFor` property (for accessibility purposes). However since our input is a custom styled component called `CheckboxInput`, we need to register it in the ESLint settings (otherwise it keeps showing an error cause it thinks there's no nested input)